src/components/MiradorMenuButton.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+
+
+/**
+ * MiradorMenuButton ~ Wrap the given icon prop in an IconButton and a Tooltip.
+ * This shares the passed in aria-label w/ the Tooltip (as title) and the IconButton
+ * All props besides icon are spread to the IconButton component
+*/
+export function MiradorMenuButton(props) {
+  const { 'aria-label': ariaLabel, disabled } = props;
+  const { icon, ...iconButtonProps } = props;
```

Comment:
I'm curious about the choice of an `icon` property instead of `children`?

Reply:
Yeah, I went back and forth on that a bit.  I would be a happy to make it required `children` instead of an explicit `icon` prop.  I was poking around some of the MUI code and saw that they have an `icon` prop in some cases (e.g. [`Tab`](https://material-ui.com/api/tab/)) (although they also have children sometimes).

Reply:
I think the thing that'd push me to `children` is because this seems like an `IconButton` wrapper, and it uses `children`. 🤷‍♂️